components/why-kedro/why-kedro.module.scss

```diff
@@ -1,10 +1,25 @@
 @import '../../styles/variables';
 @import '../../styles/mixins';
 
-.container {
+.outer {
+  background-color: $color-bg-grey;
+  padding: 5rem 0;
```

Comment:
In the design, it looks like the bottom padding needs to be bigger than the top here. Like `10rem`, though I think it looks nice as you have it.